.pre-commit-config.yaml

```diff
@@ -28,13 +28,13 @@
     - --output-format=colorized
     - |-
         --init-hook=
-        import os, platform, shutil, sys;
+        import os, platform, sys;
```

Comment:
This is unrelated.

Reply:
Sorry, @webknjaz , I need to delete it?

Reply:
You don't not introduce unrelated changes in PR.